README.md

```diff
@@ -65,6 +65,8 @@ pygmentsCodeFences = true
   description = "Personal blog."
 
 [params]
+  mainSections = ["post"]
```

Comment:
It is not needed to enter this parameter in the config since Hugo detects the sections with the most content files internally.